### ISSUE TITLE
Only 10% accuracy for scaled images!!!!!

### ISSUE THREAD

**Issue by kumarlamichhane** [naturomics/CapsNet-Tensorflow#65]
CapsNet is said to perform better for scaled images but i trained the network with original images and tested the network with the scaled images to find out the test accuracy to be only 10%... #CapsBoringNet

**Comment by GuangyaoZhang** [comment]
CapsNet is said to perform better for scaled images does not mean it can perform well without training. In my opinion, it  just has the ability of "finding " it is a scaled picture of original picture, so it is possible if you train capsnet on image with rotation angle 0, pi/2, pi than you test it on pi/3, it will perform well (pi is 3.1415...)

**Comment by TarrySingh** [comment]
Kumar — where’s your code? What did you do? 10% accuracy means you’ve potentially also messed up somewhere in your code.

**Comment by KumarLamic** [comment]
i trained the network with a training set...
i tested the network with test_data_original to get nearly 99% accuracy..
i rotated the test_data_original with 0.5degree and 1degree 
to get another test_data_rotated0p5 and test_data_rotated1
i tested the network against test_data_rotated0p5 and test_data_rotated1 to get the test accuracies around 10%.....
I cloned this repo and feed my datasets..
no clue whats going over here
